Monster.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+class Monster
+{
+private:
+	char* name;
+	size_t attackPoints;
+	size_t defensePoints;
+	
+	void copy(const Monster& other);
+	void free();
+public:
+	Monster() = default;
+	Monster(const char* name,size_t attackPoints,size_t defensePoints);
+	Monster(const Monster& other);
+
+	void setName(const char* name);
+	void setAttackPoints(size_t attackPoints);
+	void setDefensePoints(size_t defensePoints);
+
+	char* getName() const;
```

Comment:
const char*, do you know why

Reply:
because the value to which the pointer points is not going to be changed